File: Python/1020.py
n = int(input())

ano = n // 365
d = n % 365
meses = d // 30
dias = d % 30

print('{} ano(s)'.format(ano))
print('{} mes(es)'.format(meses))
print('{} dia(s)'.format(dias))

File: C/1020.c
#include <stdio.h>

int main() {
    int n, ano, d, meses, dias;

    scanf("%d", &n);

    ano = n / 365;
    d = n % 365;
    meses = d / 30;
    dias = d % 30;

    printf("%d ano(s)\n", ano);
    printf("%d mes(es)\n", meses);
    printf("%d dia(s)\n", dias);

    return 0;
}
